### PR TITLE
SetupExtensionsBuilderExtensions.RegisterLayoutRenderer supports LayoutRenderOptions

### PR DESCRIPTION
**Pull request by AlanLiu90**

### CODE REVIEW

**Comment by snakefoot** [comment]
Closing and reopening to trigger rebuild.

**Comment by snakefoot** [comment]
Close and reopen one more time, because I feel lucky

**Comment by snakefoot** [comment]
Close and reopen one more time, because I still feel lucky

File: src/NLog/SetupExtensionsBuilderExtensions.cs [hunk 1]
```diff
@@ -133,7 +134,7 @@ public static ISetupExtensionsBuilder RegisterLayoutRenderer(this ISetupExtensio
         /// <param name="layoutMethod">Callback that returns the value for the layout renderer.</param>
         public static ISetupExtensionsBuilder RegisterLayoutRenderer(this ISetupExtensionsBuilder setupBuilder, string name, Func<LogEventInfo, object> layoutMethod)
         {
-            return RegisterLayoutRenderer(setupBuilder, name, (info, configuration) => layoutMethod(info));
+            return RegisterLayoutRenderer(setupBuilder, name, (info, configuration) => layoutMethod(info), LayoutRenderOptions.None);
```
Comment (snakefoot) [inline src/NLog/SetupExtensionsBuilderExtensions.cs:137]: If you remove `LayoutRenderOptions.None` from this method, then I think you will get better code-coverage by the unit-tests.

**Comment by 304NotModified** [comment]
Green on Travis :)

**Comment by 304NotModified** [comment]
Thanks!

**Pull request merged**